package.json

```diff
@@ -9,7 +9,8 @@
     "lint": "next lint && npm run prettier-check",
     "prettier": "prettier './**/*.{js,ts,md,html,css,tsx}' --write",
     "prettier-check": "prettier './**/*.{js,ts,md,html,tsx}' --check",
-    "typescript": "tsc -p ."
+    "typescript": "tsc -p .",
+    "validate-change": "yarn build"
```

Comment:
`validate-change` should ideally be lint + typescript. Then `build` should be run separately after, any reason we aren't doing it that way? `build` is producing the assets, so should not be part of validation!

Reply:
https://github.com/stripe/stripe-connect-furever-demo/blob/4b34816cf42224d3957ed844e650517c76d932e4/package.json#L21 I had just copied it over from here, but will make that change